spec.html

```diff
@@ -128,6 +128,11 @@ <h1>Agents</h1>
         </tr>
       </table>
     </emu-table>
+
+    <emu-note>
+      <p>The value field [[AsyncContextValue]] in the list [[AsyncContextMapping]] of an Agent Record is only intended to be accessed via an [[AsyncContextKey]] field. The list is not iterable through AsyncContext APIs.</p>
+      <p>The above definition implies that, if a value in the [[AsyncContextKey]] field in an Async Context Mapping Record is not live, then its corresponding value of [[AsyncContextValue]] is not necessarily live either.</p>
```

Comment:
I think this could be better expressed in two steps:
- A live `[[AsyncContextMapping]]` does not necessarily keep its `[[AsyncContextKey]]`s alive
- For a live `[[AsyncContextMapping]]`, if an `[[AsyncContextKey]]` is not live, its corresponding `[[AsyncContextValue]]` is not necessarily kept alive.